extensions/cornerstone/src/components/ViewportDataOverlaySettingMenu/useViewportDisplaySets.ts

```diff
@@ -0,0 +1,91 @@
+import { useMemo } from 'react';
+import { useSystem } from '@ohif/core';
+import { getEnhancedDisplaySets, sortByOverlayable, DERIVED_OVERLAY_MODALITIES } from './utils';
+
+const LOW_PRIORITY_MODALITIES = ['SR', 'SEG', 'RTSTRUCT'];
+
+const sortByPriority = (a, b) => {
+  if (LOW_PRIORITY_MODALITIES.includes(a.Modality)) {
+    return 1;
+  }
+  return -1;
+};
+
+/**
+ * Hook to provide all the display sets and overlay information for a viewport
```

Comment:
Can you list the return display sets and what they mean/how they are selected?  The names help, but not quite enough to know why some display set is included or not.